kod/util/factgame/territry.kod

```diff
@@ -229,6 +230,10 @@ messages:
          return;
       }
 
+      % todo: Combine these loops because the new SetFaction accepts 
```

Comment:
Customarily "TODO" is all capitalized to make it stand out and easier to search for.